src/CMakeLists.txt

```diff
@@ -62,12 +63,13 @@ if (MSVC)
 		LINK_FLAGS "/subsystem:windows /ENTRY:mainCRTStartup")
 elseif(APPLE)
 	# Help CPack build the bundle
+	set_source_files_properties(${PROJECT_SOURCE_DIR}/asset/brd.icns
+		PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
 	set_target_properties(${PROJECT_NAME} PROPERTIES
-		RESOURCE glfw.icns
```

Comment:
Try setting `RESOURCE ${ICON}`, removing both `set_source_files_properties`'s and setting `RESOURCE DESTINATION` in `install()` to "Resources"